### DISCUSSION TITLE
add/delete hex-values in registry?

### DISCUSSION THREAD

**Discussion by RubicBG** [moudey/Shell#284] [Q&A]
hello

Is it possible with the current debug version to add hex values (or binary values) to the registry?
no information here: [https://nilesoft.org/blog/new-functions-for-handling-the-registry](url)

I tried to guess but to no avail:
`cmd=reg.set('HKCU\...', 'TargetFolderPath', '44,00,3a,00,5c,00,00,00', reg.hex) `

**Comment by RubicBG** [comment]
I struggled for three days, finally I wrote a post, 10 minutes later I found the solutions. phew

**Comment by moudey** [comment]
I will work on adding another function that allows writing a binary type.
